view/frontend/templates/catalog/product/installments.phtml

```diff
@@ -0,0 +1,9 @@
+<div>
```

Comment:
Add some div id or class property just to make it easier for who wants to customise its design. ;)